charts/groups/values.yaml

```diff
@@ -0,0 +1,14 @@
+destination:
+  name: in-cluster
+  server: ""
+
+groups:
+  - name: imaging
+    enabled: true
+    permittedRepos:
+      - https://github.com/DiamondLightSource/imaging-workflows
+    apps:
+      - name: imaging
+        repoURL: https://github.com/DiamondLightSource/imaging-workflows
```

Comment:
can we somehow avoid listing all the repos twice?

Reply:
Yeah, should be an easy-ish update to the chart. Though maybe we don't even want permitted repos and should just allow them to use whatever?